### DISCUSSION TITLE
Bigger blink build machine plans

### DISCUSSION THREAD

**Discussion by yury** [blinksh/blink#1878] [Ideas]
[8GB machine requested](https://discord.com/channels/441939332442619905/856986776187306006/1120611794621300758)

**Comment by jaidetree** [comment]
This is fascinating. Tried codespaces, my app processes kept getting reaped due to memory or cpu usage. Had to upgrade all the way to the 16 gb to get a consistent experience. Running in a 4 gb blink build image and it seems to run just fine without being reaped so far.

**Comment by carloscabanero** [comment]
Keep us posted on how it is holding up! It may be that we are running nothing by default.

**Comment by dvdsgl** [comment]
Is there anyway to get a larger home disk on the current machines?

**Reply by carloscabanero** [reply]
We don’t have larger home drives at the moment, but I’m exploring the possibilities here. In the meantime, what I do, you can tap on the ephemeral drive for node packages and all that.

PS: Problem here is not the space itself but how much would be enough given how some platforms are designed to abuse it. Ie, Python, node, ruby, etc as they try to create environments inside environments inside environments. Or in the case of Rust for example, the temporary directories grow unbounded to speed up builds, etc…
